### PR TITLE
CLIMATE-779 Add capability to run_RCMES.py

### PR DESCRIPTION
**Pull request by dlnash**
integrate ESGF for observation and model data

### CODE REVIEW

File: RCMES/configuration_files/ESGF_example/ESGF-Example.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+workdir: ./results/ESGF_example
```
Comment (lewismc) [inline RCMES/configuration_files/ESGF_example/ESGF-Example.yaml]: Can you please add an Apache license header?

**Comment by lewismc** [comment]
Hi folks, I plan to review this issue tonight and try it out. If someone else is able and willing it would be great to test out @dlnash code.
Thank you @dlnash for the contribution, it is ALWAYS real nice to have one less TODO documented within the source code :)

**Comment by lewismc** [comment]
I added the license header and I also included the import for the esgf import.

**Comment by huikyole** [comment]
Why is this pull request closed? It is not working.

**Comment by lewismc** [comment]
@huikyole anything more specific that that please?
I tried it out and was able to run the code with the provided configuration file.

**Comment by huikyole** [comment]
@lewismc Have you tested 'run_RCMES.py ./configuration_files/ESGF_example/ESGF-Example.yaml'?
It does not work, and the goal of this pull request is making run_RCMES.py work with ESGF-Example.yaml.

**Pull request merged**